Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.gitignore
rename from .idea/.gitignore
rename to .gitignore
--- a/.idea/.gitignore	
+++ b/.gitignore	
@@ -1,8 +1,9 @@
 # 디폴트 무시된 파일
-/shelf/
+/.idea/shelf/
 /workspace.xml
 # 에디터 기반 HTTP 클라이언트 요청
 /httpRequests/
 # Datasource local storage ignored files
 /dataSources/
 /dataSources.local.xml
+out/
\ No newline at end of file
